installer/mod.ts

```diff
@@ -153,7 +153,7 @@ async function generateExecutable(
   filePath: string,
   commands: string[]
 ): Promise<void> {
-  commands = commands.map((v): string => '"' + v + '"');
+  commands = commands.map((v): string => `"${v.replace(/\"/g, '\\"')}"`);
```

Comment:
Hrm... I don't think this is sufficient to escape the strings properly. (I can't immediately think of a counter example...) What about using JSON.stringify() on the strings?

Reply:
yes, it works.

Reply:
You can leave out the anonymous function I think.

Reply:
@ry No, it can't. typescript will throw an error.